src/test/resources/config.properties

```diff
@@ -0,0 +1 @@
+mongodb+srv://at-sso-api:4gil3th0ught@cluster0.vliih.mongodb.net/at-sso-db?retryWrites=true&w=majority
```

Comment:
We don't need this sensitive info here, if needed, please don't commit it

Reply:
Okey, I will put the file on the gitignore